src/models/src/blocks/view.rs

```diff
@@ -80,4 +80,27 @@ pub struct SlackViewStateValue {
     #[serde(rename = "type")]
     pub action_type: SlackActionType,
     pub value: Option<String>,
+    pub selected_date: Option<String>,
+    pub selected_time: Option<String>,
+    pub selected_conversation: Option<SlackConversationId>,
+    pub selected_channel: Option<SlackChannelId>,
+    pub selected_user: Option<SlackUserId>,
+    pub selected_option: Option<SlackViewStateValueSelectedOption>,
+    pub selected_conversations: Option<Vec<SlackConversationId>>,
+    pub selected_users: Option<Vec<SlackUserId>>,
+    pub selected_options: Option<Vec<SlackViewStateValueSelectedOption>>,
+}
+
+#[skip_serializing_none]
+#[derive(Debug, PartialEq, Clone, Serialize, Deserialize, Builder)]
+pub struct SlackViewStateValueSelectedOption {
+    pub text: SlackViewStateValueSelectedOptionText,
+    pub value: String,
+}
+
+#[skip_serializing_none]
+#[derive(Debug, PartialEq, Clone, Serialize, Deserialize, Builder)]
+pub struct SlackViewStateValueSelectedOptionText {
```

Comment:
This (painfully) resembles `SlackBlockPlainText` - isn't the same type? Maybe we can just reuse it?

Reply:
Replaced it with `SlackBlockPlainText` @ 1cc200b .